integration/mesos/src/main/java/alluxio/mesos/AlluxioFramework.java

```diff
@@ -97,8 +97,14 @@ public void offerRescinded(SchedulerDriver driver, Protos.OfferID offerId) {
     }
 
     @Override
-    public void registered(SchedulerDriver driver, Protos.FrameworkID frameworkId,
+    public void registered(final SchedulerDriver driver, Protos.FrameworkID frameworkId,
```

Comment:
The scheduler is accessed in the anonymous shutdown hook below, so it needs to be final